vault/dendron.tutorial.navigation-basics.md

```diff
@@ -2,47 +2,55 @@
 id: gHdxXlNMr1w4xqee0n-Mb
 title: Navigation Basics
 desc: 1. Navigation Basics
-updated: 1625573454967
+updated: 1639194721158
 created: 1625563862198
 nav_order: 0
 ---
 
 Let's do a brief overview on how to navigate the Dendron UI.
 
-_Quick note on running commands in VS Code:_
+_Quick note on running commands in VSCode:_
 
->If you're unfamiliar with VS Code, the command palette is used to quickly run commands, including the commands for Dendron. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. Throughout the tutorial, if you see instructions telling you to run a command, remember to bring up the command palette and then type the command name to run it.
+>If you're unfamiliar with VSCode, the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) is used to quickly run commands, including the commands for Dendron. To bring up the command palette, use `Ctrl+Shift+P` / `Cmd+Shift+P` and start typing the command. Throughout the tutorial, if you see instructions telling you to run a command, remember to bring up the command palette and then type the command name to run it.
 
-![Basic UI](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-layout.png)
+![[dendron://dendron.dendron-site/asset.preview#tutorial-dendron-layout,1:#*]]
 
+> 💡 VSCode lets you re-position any of these windows as you'd like, so feel free to rearrange the windows into the view that works best for you!
 #### 1. Editor Pane
 
-This is where you can write and edit your notes. Notes in Dendron are Markdown files.
+This is where you can write and edit your notes. **Notes** in Dendron are Markdown files (ex. `my.note.md`).
 
 #### 2. Preview Pane
 
 This shows the rendered Markdown of what your currently opened note looks like. If you close this pane, you can always bring it back by opening the command palette `Ctrl+Shift+P` / `Cmd+Shift+P` and running the `Dendron: Show Preview` command.
 
->💡 The preview pane is read-only and cannot be used to edit notes. All editing must be done in the editor pane.
+>💡 The preview pane is **read-only** and cannot be used to edit notes. All editing must be done in the editor pane.
 
 #### 3. Dendron Workspace
 
-This shows the files in your vault. A vault is where Dendron stores your markdown notes, assets, and some other helper files. Clicking on a note here will open it into the editor pane.
+This shows the files in your vault. A **vault** is where Dendron stores your Markdown notes, assets (such as images), and some other helper files. Clicking on a note here will open it into the editor pane.
```

Comment:
lets add a link to [[dendron.topic.vault]]

Reply:
`stores your notes and assets (eg. images, pdfs, etc)`.

Reply:
> Clicking on a note here will open it into the editor pane.

Remove this sentence. "We discourage folks from using the workspace panel but instead use the [[tree view|dendron.tutorial.navigation-basics#6-tree-view]] to navigate notes"